language/functional_tests/src/tests/global_config_tests.rs

```diff
@@ -27,7 +27,11 @@ fn parse_account_positive() {
 
 #[test]
 fn parse_account_negative() {
```

Comment:
Also add a positive test case? Just to make sure "//! account: alice, 1, 2, validator" parses.